pulpcore/cli/common/context.py

```diff
@@ -39,6 +39,7 @@
     core="pulpcore",
     deb="pulp_deb",
     file="pulp_file",
+    python="pulp_python",
```

Comment:
Without that, the commands never worked with pulpcore <= 3.10, right?

Reply:
Yeah.